src/root.component.tsx

```diff
@@ -26,15 +26,21 @@ function Root(props) {
           showPatientSummary={setShowPatientSummary}
         />
       </Route>
-      <Route
-        path="/patient/:patientUuid/chart"
-        exact
-        component={PatientChartSummary}
-      />
-      <Route
-        path="/patient/:patientUuid/chart"
-        render={routeProps => <LevelTwoRoutes match={props.match} />}
-      />
+      <main className="omrs-main-content">
+        {showPatientSummary && (
+          <Route
+            path="/patient/:patientUuid/chart"
+            exact
+            component={PatientChartSummary}
+          />
+        )}
+        {showPatientSummary && (
```

Comment:
I was under the impression that the level two routes are there own pages that don't require clicking on the banner first in order to see them. I do not think we should only show the level two routes if you've clicked on the banner.

Reply:
Level two routes are their own pages and visible by default. Clicking on the banner hides them in order to display the profile.